tidb-server/server/server.go

```diff
@@ -63,6 +63,20 @@ func (s *Server) releaseToken(token *Token) {
 	s.concurrentLimiter.Put(token)
 }
 
+// Generate a random string using ASCII characters but avoid seperator character.
+// See: https://github.com/mysql/mysql-server/blob/5.7/mysys_ssl/crypt_genhash_impl.cc#L435
+func randomBuf(size int) []byte {
+	buf := make([]byte, size)
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
Call Seed once is fine. I think you may call it when in initialization function of package .